<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>9</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 41, 45);</string>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_log">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_percentage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_division">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mult">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sum">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_equal">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decimal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 112, 255);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_change_sing">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(248, 49, 255);</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_power">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>xยฒ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tan">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cot">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>cot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cos">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>